### PR TITLE
Sunday package bumpity bump

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ flaky==3.7.0
 
 
 # To test google spreadsheet uploading
-google-api-python-client==2.65.0
+google-api-python-client==2.80.0
 google-auth-httplib2==0.1.0
-google-api-core==1.32.0  # temporarily due to protobuf conflictwith web3
-google-auth==1.25.0  # temporarily due to protobuf conflictwith web3
+google-api-core==2.11.0  # temporarily due to protobuf conflictwith web3
+google-auth==2.16.2  # temporarily due to protobuf conflictwith web3
```
Comment (yabirgb) [inline requirements_dev.txt:21]: Guess we can remove the comments then?

File: rotkehlchen/externalapis/etherscan.py [hunk 2]
```diff
@@ -355,7 +355,7 @@ def get_transactions(
                 options['txHash'] = period_or_hash.hex()
                 parent_tx_hash = period_or_hash
 
-        transactions: Union[Sequence[EvmTransaction], Sequence[EvmInternalTransaction]] = []  # noqa: E501
+        transactions: Union[list[EvmTransaction], list[EvmInternalTransaction]] = []  # type: ignore  # noqa: E501
```
Comment (yabirgb) [inline rotkehlchen/externalapis/etherscan.py:358]: why this change?

Comment (LefterisJP) [reply rotkehlchen/externalapis/etherscan.py:358]: because this was the least type ignores to make it work. list[None] != list[X] in mypy

File: requirements_dev.txt (hunk 1 above)
Comment (LefterisJP) [reply requirements_dev.txt:21]: in another PR I guess
